v2/clients/http/utils/common.go

```diff
@@ -88,6 +91,34 @@ func createRequestWithRawData(ctx context.Context, httpMethod string, url string
 	return req, nil
 }
 
+// createRequestFromYamlFilePath creates multipart/form-data request with YAML file
+func createRequestFromYamlFilePath(ctx context.Context, httpMethod string, url string, yamlFilePath string) (*http.Request, errors.EdgeX) {
```

Comment:
This isn't YAML file specific, it code by any file correct? Rename to `createRequestFromFilePath`

Reply:
Yes, it is. Renamed.